tests/cases/standalone/common/ttl/ttl_instant.sql

```diff
@@ -0,0 +1,145 @@
+CREATE TABLE test_ttl(
+       ts TIMESTAMP TIME INDEX,
+       val INT,
+       PRIMARY KEY (`val`)
+) WITH (ttl = 'instant');
+
+SHOW CREATE TABLE test_ttl;
+
+INSERT INTO
+       test_ttl
+VALUES
+       (now(), 1),
+       (now(), 2),
+       (now(), 3);
+
+SELECT
+       val
+from
+       test_ttl;
```

Comment:
non-blocking: Consider adding order val or sqlness sort result to ensure result is stable.